packages/serverless-contracts/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.18.3",
+    "aws-lambda": "^1.0.7",
```

Comment:
We shouldn't need this. I tried https://github.com/swarmion/swarmion/pull/147, can you rebase on it when it is merged?

Reply:
i need to do the same in this package but yes i will try it